db/migrations/versions/f783496def21_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: f783496def21
+Revises: 5ce0b8e0e1b6
+Create Date: 2022-12-01 14:59:04.914461
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'f783496def21'
+down_revision = '5ce0b8e0e1b6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('just_scores',
+    sa.Column('score_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('score', sa.Integer(), nullable=False),
+    sa.Column('justification', sa.Text(), nullable=False),
+    sa.Column('application_id', postgresql.UUID(), nullable=True),
+    sa.Column('date_created', sa.DateTime(), nullable=True),
```

Comment:
🤔 Should this have a default? like func.now()?

Reply:
i thought it was weird it didn't show here... it is func.now() in the model 👍🏾 not sure why there's nothing that shows this in the migration, have tested this and date_created does get auto generated 👌🏾

Reply:
have you got it set as the `server_default`?

Reply:
Ah, I see @SentientHousePlant's point, we can set a default in the migration itself e.g. as per https://github.com/communitiesuk/funding-service-design-application-store/blob/559d62e7f08f1dc5b143283f2110fc46014803d3/db/models/applications.py#L41

Reply:
ahh thank you @SentientHousePlant @gidsg that makes sense, all done now 👌🏾